articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,171 @@
+ - **Switched Virtual Interface(SVI)** refers to a virtual interface which connects vlans on the network devices to their respective router engines.
+ - **Default Gateway** is a device  that forwards IP packets and provides an access point to the other networks. For instance, a router.
+ - **Access Point** is a network device that connects to a router or switch , creates a wireless local area network and act as the link between the router and the network users.
+ - **Switch port** refers to a physical opening on the switch or router where ethernet cables can be plugged.
+ - **Router Interface** refers to a path that enables the router to connect to the network.
+
+### What is Inter VLAN routing? <a name="intervlanrouting"></a>
+Virtual LANs (`VLANS`) are networks segments on a switched LAN. **Inter vlan routing** this refers to the movement of packets across the network between  hosts in different network segments.``VLANS`` make it easier for one to segment a single network, they improve the performance of the network and are flexible as they base on logical connections. VLANS act as separate subnets on the network. For that reason to move packets from one VLAN to another and enable communications between hosts we have to configure the network to enable inter-vlan routing.
+
+### Inter-VLAN routing methods. <a name="methods"></a>
+#### Legacy inter-vlan routing
+In this solution multiple router interfaces were used, each connecting to a switch port in different vlans. The interfaces served as default gateways, and this required additional cabling when the network was to be expanded. Hence becoming more costly for the network cables and infrastructure.
+
+#### Router-on-a-stick
+In this method, unlike the legacy, one physical interface port is used for routing traffic between the network segments. The network administrator does not require creating all the vlan interface like fa0/1 to fa0/10, all the interfaces from 1 to 10 are created in a single interface. This method is simple to implement and use for smaller to medium-sized organisation networks.
+#### Layer 3 switch using switched virtual interface(SVIs)
+Currently, this method of inter-VLAN routing that uses layer 3/multilayer switch and Switched Virtual Interfaces (SVI) are being used more commonly. SVIs are created for VLANs that exist on the switch and they perform the same function for the VLANs as that of a router.
+Layer 3 switches are expensive, but suitable for large organisation networks.
+
+### Network configurations for Inter-VLAN communication using Router-On-Stick method
+In this article, we will learn how to configure inter-VLAN routing using the router-on-a-stick method.
+
+Consider a LAN with 4 PCs, 1 switch, and a router connected as shown in the image:
+
+![Local Area Network](/engineering-education/inter-vlan-routing/network.jpg)
+
+We want to configure two ``VLANS 10`` and ``20``. ``PC0`` and ``PC1`` on ``VLAN10`` while ``PC2`` and ``PC3`` are on ``VLAN20``.
```

Comment:
Adding a single ` (back quote) at the start and end of the word, would highlight it. Remove two backquotes.

I want something like `VLAN 10`

Reply:
Make the changes wherever necessary